DEV-4/DEV-4/DescriptionEntity.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace DEV_4
+{
+    /// <summary>
+    /// Class for containing general information of entities (GUID and Description).
+    /// </summary>
+    class DescriptionEntity
+    {
+        public string description;
+        public string id;
+        public string Description
+        {
+            get
+            {
+                return description;
```

Comment:
If the body of the get or set method of a property consists of a single statement, the statement is
written on the same line as the method signature. White space is inserted between the property
method (get, set) and the opening brace. This will create visually more compact class definitions..